src/main/java/io/redlink/more/data/service/GarminService.java

```diff
@@ -359,6 +380,136 @@ private void addAuthorizationHeader(ApiClient apiClient, UserAccessTokenWithData
         );
     }
 
+
+    private Set<String> userIdsFromData(Map<String, List<Map<String, Object>>> data) {
```

Comment:
kann es echt mehrere UserIds in den Daten geben? Wenn nicht würde ich da eher nur einen String zurück geben und anstelle von `.collect(..)` ein `findFirst(..)` nehmen

Reply:
haben geredet. Die schicken Daten für alle User in einen Request